target/linux/realtek/dts-5.10/rtl8393_netgear_gs750e.dts

```diff
@@ -0,0 +1,281 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "rtl839x.dtsi"
+
+#include <dt-bindings/input/input.h>
+#include <dt-bindings/gpio/gpio.h>
+
+/ {
+	compatible = "netgear,gs750e", "realtek,rtl8393-soc";
+	model = "Netgear GS750E";
+
+	aliases {
+		label-mac-device = &ethernet0;
+	};
+
+	memory@0 {
+		device_type = "memory";
+		reg = <0x0 0x8000000>;
+	};
+
+	keys {
+		compatible = "gpio-keys";
+
+		reset {
+			label = "reset";
+			gpios = <&gpio0 11 GPIO_ACTIVE_LOW>;
+			linux,code = <KEY_RESTART>;
+		};
+	};
+
+	virtual_flash {
+		compatible = "mtd-concat";
+
+		devices = <&fwconcat0 &fwconcat1 &fwconcat2>;
```

Comment:
I found a lot of devices (if not all) using my devices definition: https://github.com/openwrt/openwrt/search?q=devices+%3D+%3C%26fwconcat0 Which is the right one?

Reply:
It seems this should be a list of node references, not a node reference with two extra cells. See for example: https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/commit/?id=25e4f5220efead592c83200241e098e757d37e1f

In the end, the resulting binary device tree (DTB) will contain exactly the same bytes for both notations. There also isn't any formal upstream binding you could check against, because AFAICT it was never accepted.

Reply:
Ok then, I will fix it! :wink: